recipe/meta.yaml

```diff
@@ -9,7 +9,7 @@ source:
   sha256: 5d427fddd6d0d33af8c16bbb08cd9818514a518048b4719ce6bbedc3d79640c0
 
 build:
-  number: 1
+  number: 4
```

Comment:
This can be 2. It is confusing to jump build numbers, users may ask, where did build numbers 2-3 go?

Reply:
Oops I think that was an artifact from messing around. Moved it back to 2.